certificateshop-logic/src/main/java/com/epam/esm/configuration/LogicConfig.java

```diff
@@ -23,6 +27,12 @@
  * The LogicConfig class.
  */
 @Configuration
+@EnableJpaRepositories(basePackages = {
+        "com.epam.esm.repository"
+})
+@EntityScan(basePackages = {
+        "com.epam.esm.model"
+})
```

Comment:
You have only one package, you can use `@EntityScan("com.epam.esm.model")`. Check if it's good for EnableJpaRepositories annotation as well

Reply:
Done